### DISCUSSION TITLE
Superset Chart duplicating

### DISCUSSION THREAD

**Discussion by Prathushagoud** [apache/superset#28329] [Q&A / Help]
Hi all,
Can anyone please help to duplicate a chart in the dashboard, Im facing problem to duplicate the chart .
In the same dashboard with same dimensions, just change a single variable .
i am to creating a new chart everytime, i want duplication of the chart

**Comment by gpandolfino** [comment]
put +1 here https://github.com/apache/superset/discussions/18693
